cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -111,6 +111,7 @@ func TestValidateNodeRegistrationOptions(t *testing.T) {
 		{"valid-nodename", kubeadmapiv1beta2.DefaultUrlScheme + "://" + "/some/path", false}, // supported, with socket url
 		{"valid-nodename", "bla:///some/path", true},                                         // unsupported url scheme
 		{"valid-nodename", ":::", true},                                                      // unparseable url
+		{"valid-nodename", "", false},                                                        // invalid CRISocket (path is not absolute)
```

Comment:
hi, if the path is not absolute an error should be returned:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/validation/validation.go#L535

why aren't we expecting an error (false)?

Reply:
hmm, weird. Maybe the changes in my workspace didn't get staged correctly. This should be true (expecting error). I must have changed it to false to simulate a failed test and that accidentally got staged.
Sorry about that. Updated and force-pushed.